pyfolio/timeseries.py

```diff
@@ -1024,6 +1024,28 @@ def gen_drawdown_table(returns, top=10):
     return df_drawdowns
 
 
+def rolling_volatility(returns, rolling_vol_window):
+    """
+    Determines the rolling volatility of a strategy.
+
+    Parameters
+    ----------
+    returns : pd.Series
+        Daily returns of the strategy, noncumulative.
+         - See full explanation in tears.create_full_tear_sheet.
+    rolling_vol_window : int
+        Length of rolling window, in days, over which to compute.
+
+    Returns
+    -------
+    pd.Series
+        Rolling volatility.
+    """
+
+    return returns.rolling(rolling_vol_window).std() \
```

Comment:
My rolling_volatility takes in the returns and the window as an integer. It returns a pd.Series.

Empyrical.annual_volatility also takes in the returns, but no window over which to compute the vol, since it does not return a pd.Series, but instead a float. Empyricals annual vol takes in a period as a string over which period to annualize, e.g., daily, weekly, ...

The only way I could use empyrical.annual_volatility in my function is, if I split the returns into sublists of length "rolling_vol_window" and compute empyrical.annual_volatility over each of those, which makes it really ugly and maybe also slow.

If you have a nicer idea on how to use empyrical, please tell me. You are more familiar with that, than I.